release/nightly_tests/many_nodes_tests/actor_test.py

```diff
@@ -43,7 +38,15 @@ def main():
         sleep(10)
         return
     actor_ready_start = perf_counter()
-    test_actor_ready(actors)
+    total_actors = len(actors)
+    objs = [actor.foo.remote() for actor in actors]
+
+    while len(objs) != 0:
+        objs_ready, objs = ray.wait(objs, timeout=10)
```

Comment:
any reason why we change this back?

Reply:
It's not changing anything back I believe. The last change is 1 year ago.
I put this to show logs for debug.

Reply:
(Oh, I think you might mistake this with the other changes we did recently)